gradle-suppressible-error-prone/src/main/java/com/palantir/gradle/suppressibleerrorprone/modes/interferences/SuppressingAndApplyingInterference.java

```diff
@@ -43,6 +43,6 @@ public CommonOptions interfere(Map<ModeName, CommonOptions> modeCommonOptions) {
         return suppress.naivelyCombinedWith(apply)
                 .withExtraErrorProneCheckFlag(
                         "SuppressibleErrorProne:PreferPatchChecks",
-                        apply.patchChecks().asCommaSeparated().orElse(""));
```

Comment:
This was not evaluated lazily before.

Reply:
Calling `patchChecks()` is the issue here - so that needs be evaluated lazily. The value of flag is calculated from that so also needs to be lazy.